Packs/CortexXDR/ReleaseNotes/4_1_11.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Palo Alto Networks Cortex XDR - Investigation and Response
+- Fixed an issue where the integration did not handle mirroring properly.
```

Comment:
can you specify in which cases? so the customer will know if it is relevant for him to upgrade the pack.
```suggestion
- Fixed an issue where the integration did not handle mirroring properly.
```

Reply:
There was a problem with the incidents update time, I am not sure we should get too specific, what do you think?

Reply:
usually you want to be specific.